tools/licenses/lib/main.dart

```diff
@@ -1889,7 +1889,7 @@ class _RepositoryBoringSSLDirectory extends _RepositoryDirectory {
   _RepositoryDirectory createSubdirectory(fs.Directory entry) {
     if (entry.name == 'src')
       return _RepositoryBoringSSLSourceDirectory(this, entry);
-    return super.createSubdirectory(entry);
+    return _RepositoryBoringSSLDirectory(this, entry);
```

Comment:
this looks strange - what is the meaning of this?

Reply:
It means that all subdirectories in //third_party/boringssl (except src) are licensed like //third_party/boringssl itself, rather any becoming _RepositoryGenericThirdPartyDirectory.

Reply:
hmm.. what if you have another `src` folder nested deeper in boringssl? With this it will be treated as top-level `src`, which is rather strange.